ext/node/polyfills/worker_threads.ts

```diff
@@ -243,7 +253,17 @@ class NodeWorker extends EventEmitter {
         this.emit("messageerror", err);
         return;
       }
-      this.emit("message", message);
+      if (
+        // only emit "online" event once, and since the message
+        // has to come before user messages, we are safe to assume
+        // it came from us
```

Comment:
Is it guaranteed that this will arrive before user messages? In the past we had trouble with raciness between user and internal messages

Reply:
Hmm I mean this gets called as part of bootstrap. Is it possible for user code to execute before bootstrap?

Reply:
It shouldn't. Let's try it out first. Maybe I'm overthinking.